cmd/up.go

```diff
@@ -54,6 +56,7 @@ var upCmd = &cobra.Command{
 }
 
 func init() {
+	upCmd.Flags().BoolVar(&UpInsecureRepo, "insecure-repository", false, "Use Insecure Repository")
```

Comment:
@surajssd , any suggestion for me to improve?

Right now, it only impacts convertion of image stream for openshift.